data/_models/shared/Processes.csv

```diff
@@ -1,119 +1,143 @@
+Individual consent,,data use permissions,ontology,,,CatalogueOntologies,Data use permissions,,,,,http://purl.obolibrary.org/obo/DUO_0000001,"A data item that is used to indicate consent permissions for datasets and/or materials, and relates to the purposes for which datasets and/or material might be used.","Patient registry, FAIR Genomes"
+Individual consent,,data use modifiers,ontology_array,,,CatalogueOntologies,Data use modifiers,,,,,http://purl.obolibrary.org/obo/DUO_0000017,"Data use modifiers indicate additional conditions for use. For instance, a dataset is restricted to investigations into specific diseases or performed at specific geographical locations.","Patient registry, FAIR Genomes"
+Individual consent,,data use specification,text,,,,,,,,,http://semanticscience.org/resource/SIO_000090,"Further specification of applied data use permissions and modifiers. For example, a list of countries in case of geographic restrictions or a list of diseases when restricted to disease-specific research.","Patient registry, FAIR Genomes"
+Individual consent,,allow recontacting,ontology_array,,,CatalogueOntologies,Allow recontacting,,,,,http://purl.obolibrary.org/obo/NCIT_C25737,The procedure of recontacting the patient for specified reasons. This means the patient agrees to be re-identifiable under those circumstances.,"Patient registry, FAIR Genomes"
+Observations,Processes,,,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C25398,Findings and circumstances relating to the examination and treatment of a individual,"Patient registry, FAIR Genomes"
+Observations,,date of observation,date,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C119798,The date on which an observation took place,"Patient registry, FAIR Genomes"
+Observations,,age at observation,period,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C192236,"The age of the individual at the time of observation (e.g., ""P55Y8M12D"")",Patient registry
+Observations,,type of visit,ontology,,,CatalogueOntologies,Visit type,,,,,http://purl.obolibrary.org/obo/NCIT_C103165,"The kind of visit undertaken by the individual within a study such as inpatient, outpatient, telephone, baseline, follow up, etc.","Patient registry, FAIR Genomes"
+Observations,,location of observation,ref,,,CatalogueOntologies,Organisations,,,,,http://purl.obolibrary.org/obo/NCIT_C119804,The site in or on the body at which the observation is made.,Patient registry
+Sequencing runs,Processes,,,,,,,,,,,http://edamontology.org/topic_3168,"The determination of complete (typically nucleotide) sequences, including those of genomes (full genome sequencing, de novo sequencing and resequencing), amplicons and transcriptomes. In FAIR Genomes: Sequencing. Ideally: SequencingRuns.","Patient registry, FAIR Genomes"
```

Comment:
I thought you wanted to merge all these details also into Sample? That is probably why I started to talk about 'Assay' assuming we gonna merge Sample and all processes around it into one Sequencing assay (which then behaves as the Assay in ISA-TAB https://isa-specs.readthedocs.io/en/latest/isamodel.html lumping all info on the materials etc together. But maybe it is just me taking the change of model we recently started too far.

Reply:
I see there is a lot of details so I do undrestand.